### PR TITLE
[LILA-6854] Address Team Omega comments, update emails and versions

### PR DESCRIPTION
**Pull request by 0xmichalis**
* Address feedback from https://github.com/OmegaAudits/toucan-dynamic-fee-pools/pull/2
* Update security emails
* Bump versions
* Optimize zero amount checks

### CODE REVIEW

**Comment by lirona** [comment]
there's no reason to check `requestedAmount > 0` since it's a `uint256`, more efficient to check `requestedAmount != 0`
in `_calculateFee`
https://github.com/ToucanProtocol/dynamic-fee-pools/blob/9732dc6eb510876157c6a7c62511005417ffebe5/src/FeeCalculator.sol#L404C17-L404C36

**Review by lirona — COMMENTED** [review]
i have 1 comment that wasn't in the audit, other than that lgtm

**Comment by 0xmichalis** [comment]
> there's no reason to check requestedAmount > 0 since it's a uint256, more efficient to check requestedAmount != 0
in _calculateFee

This is true for all uint256 > 0 checks in the contracts. Updated

**Review by lirona — APPROVED** [review]

File: src/FeeCalculator.sol [hunk 1]
```diff
@@ -2,7 +2,7 @@
 //
 // SPDX-License-Identifier: UNLICENSED
 
-// If you encounter a vulnerability or an issue, please contact <info@neutralx.com>
+// If you encounter a vulnerability or an issue, please contact <security@toucan.earth> or visit security.toucan.earth
```
Comment (aspiers) [inline src/FeeCalculator.sol:5]: https://security.toucan.earth/ would be better here.

**Review by aspiers — COMMENTED** [review]
Retro +1

**Pull request merged**